src/views/SearchWikipedia.vue

```diff
@@ -18,10 +17,10 @@
       name: 'SearchWikipedia',
       components: { ListView },
       methods: {
-        ...mapActions(['update', 'search', 'clear']),
-        onSubmit: function(e) {
+        ...mapActions(['search', 'clear']),
+        onInput: function(e) {
           e.preventDefault()
-          this.search()
+          this.search(e.target.value)
```

Comment:
You should trim the value here

Reply:
I didn't trim the value here but trim during the ajax url and some checker, in addition i add the utils/api, that can be used for the next ticket. https://github.com/wikimedia/wikistories-prototype/pull/4/commits/bfd01e8c90f42478f9f3d2bd84af49ebbb7eaf16